app/controllers/posts_controller.rb

```diff
@@ -24,4 +24,10 @@ def create
       redirect_to '/posts/new'
     end
   end
+
```

Comment:
You should use **CanCanCan** for Post authorization in `Post Controller`:
```ruby
class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```
You may check out [this link](https://github.com/CanCanCommunity/cancancan#controller-helpers) to learn about that.